src/index.js

```diff
@@ -38,6 +37,7 @@ import { SettingsProvider } from './SettingsProvider';
 const { LIST_TYPES } = consts;
 
 const MainAppWithApolloProvider = () => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
```

Comment:
@ardasnturk please have a few checks on your side, if everything works as before?!

Reply:
i have not encountered any problems. i can say that it works well. 👍